src/group_theory/group_action/basic.lean

```diff
@@ -0,0 +1,212 @@
+/-
+Copyright (c) 2018 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes
+-/
+import group_theory.group_action.defs
+import group_theory.group_action.group
+import group_theory.coset
+
+/-!
+# Basic properties of group actions
+-/
```

Comment:
The main reason this file is separate from `group_theory.group_action.group` is that the `coset` import pulls in quite a deep tree.

Reply:
Do you mind expanding the module doc with the main definitions and theorems?

Reply:
I don't really have the background to pick out the "main" ones - so I'd be inclined to just put this in, and let someone with more background fill out the docs

Reply:
OK, no worries.
bors r+